src/backend/database/factories/ClusterFactory.php

```diff
@@ -27,7 +27,7 @@ public function definition(): array {
 
         return [
             'name' => 'Cluster '.$faker->county(),
-            'geo_data' => '{"leaflet_id":416,"type":"manual","geojson":{"type":"Polygon","coordinates":[[[-1.0021831137920607,34.09735878800838],[-1.0037278294879668,34.08104951280351],[-0.9961758791705448,34.08001945331692],[-0.9831315606570004,34.079332746992485],[-0.9745497443261169,34.08036280647911],[-0.9889671831741885,34.09890387723834]]]},"display_name":"My Cluster","selected":true,"draw_type":"draw","lat":-0.991455885101313,"lon":34.08617119747313}',
+            'geo_json' => '{"type":"Polygon","coordinates":[[[-1.0021831137920607,34.09735878800838],[-1.0037278294879668,34.08104951280351],[-0.9961758791705448,34.08001945331692],[-0.9831315606570004,34.079332746992485],[-0.9745497443261169,34.08036280647911],[-0.9889671831741885,34.09890387723834]]]}',
```

Comment:
This is not a valid GeoJSON. 

<img width="362" height="103" alt="Image" src="https://github.com/user-attachments/assets/7ce66021-0e5f-4d33-bbb5-0c4b23af31b0" />